test/sass/typograpy.spec.scss

```diff
@@ -159,3 +164,118 @@
   }
 
 }
+
+@include describe('@mixin label-as-content') {
+
+  @include it('generates defaults for label-as-content') {
+
+    @include assert() {
+
+      @include output {
+        @include typography.label-as-content();
+      }
+
+      @include expect {
+        color: $text-secondary;
+        letter-spacing: 0.5px;
+        font-family: "Noto Sans", Arial, Helvetica, sans-serif;
+        font-size: 0.6875rem;
+        font-weight: 400;
+        line-height: 2.1818181818;
+        text-transform: uppercase;
+      }
+
+    }
+
+  }
+
+  @include it('generates different color for text and default text-transform') {
+
+    @include assert() {
+
+      @include output {
+        @include typography.label-as-content($primary);
```

Comment:
Set as an arbitrarily named local variable e.g. `$_new-color` just within this `assert` block.